packages/garbo/src/familiar/freeFightFamiliar.ts

```diff
@@ -42,7 +42,10 @@ const DEFAULT_MENU_OPTIONS = {
   allowAttackFamiliars: true,
   mode: "free",
 } as const;
-export function menu(options: MenuOptions = {}): GeneralFamiliar[] {
+export function menu(
+  options: MenuOptions = {},
+  fight?: string,
```

Comment:
Instead of doing this, add an `excludeFamiliar: Familiar[]` parameter to `MenuOptions`